Manifests/ManifestsApple/com.apple.MCX-TimeServer.plist

```diff
@@ -145,33 +149,27 @@ The payload organization for a payload need not match the payload organization i
 		</dict>
 		<dict>
 			<key>pfm_description</key>
-			<string>The ntp server to connect to.</string>
+			<string>The NTP host to connect to. Separate multiple hosts with commas</string>
 			<key>pfm_description_reference</key>
-			<string>The ntp server to connect to.
-Availability: Available in macOS 10.12.4 and later.</string>
+			<string>The ntp server to connect to. Availability: Available in macOS 10.12.4 and later.</string>
 			<key>pfm_name</key>
 			<string>timeServer</string>
 			<key>pfm_title</key>
 			<string>Time Server</string>
 			<key>pfm_type</key>
 			<string>string</string>
-			<key>pfm_macos_min</key>
-			<string>10.12.4</string>
```

Comment:
Time Server payload in Apple config profile reference (https://developer.apple.com/business/documentation/Configuration-Profile-Reference.pdf) includes this info, so would like to keep the macOS min data

Reply:
I think this was a per-property leftover from the old undivided MCX manifest before it was split into subdomains. Since both keys in this subdomain have the same requirement, we moved it to the root of the manifest on lines 26-27.